builtin/providers/google/resource_storage_bucket.go

```diff
@@ -0,0 +1,154 @@
+package google
+
+import (
+	"errors"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+
+	"google.golang.org/api/storage/v1"
+)
+
+func resourceStorageBucket() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceStorageBucketCreate,
+		Read:   resourceStorageBucketRead,
+		Update: resourceStorageBucketUpdate,
+		Delete: resourceStorageBucketDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"predefined_acl": &schema.Schema{
+				Type:     schema.TypeString,
+				Default:  "private",
+				Optional: true,
+				ForceNew: true,
+			},
+			"location": &schema.Schema{
```

Comment:
This somehow defaults to "US" anyway. I am not sure where terraform gets that from. Similarly, AWS_S3 bucket provider doesn't set default location. Would be great to get some terraform feedback on this. Nevertheless, it can be also easily changed to default to "US"...